en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md

```diff
@@ -0,0 +1,262 @@
+                <property expression="json-eval($.tags)" name="tags" scope="default" type="STRING"/>
+                <property expression="json-eval($.timeout)" name="timeout" scope="default" type="STRING"/>
+                <property expression="json-eval($.mode)" name="mode" scope="default" type="STRING"/>
+                <property expression="json-eval($.securityGroupIds)" name="securityGroupIds" scope="default" type="STRING"/>
+                <property expression="json-eval($.subnetIds)" name="subnetIds" scope="default" type="STRING"/>
+                <amazonlambda.init>
+                    <region>{$ctx:region}</region>
+                    <accessKeyId>{$ctx:accessKeyId}</accessKeyId>
+                    <secretAccessKey>{$ctx:secretAccessKey}</secretAccessKey>
+                    <blocking>{$ctx:blocking}</blocking>
+                </amazonlambda.init>
+                <amazonlambda.createFunction>
+                    <functionName>{$ctx:functionName}</functionName>
+                    <functionDescription>{$ctx:functionDescription}</functionDescription>
+                    <apiVersionCreateFunction>{$ctx:apiVersionCreateFunction}</apiVersionCreateFunction>
+                    <s3Bucket>{$ctx:s3Bucket}</s3Bucket>
+                    <s3Key>{$ctx:s3Key}</s3Key>
+                    <s3ObjectVersion>{$ctx:s3ObjectVersion}</s3ObjectVersion>
+                    <zipFile>{$ctx:zipFile}</zipFile>
+                    <targetArn>{$ctx:targetArn}</targetArn>
+                    <environmentVariables>{$ctx:environmentVariables}</environmentVariables>
+                    <handler>{$ctx:handler}</handler>
+                    <kmsKeyArn>{$ctx:kmsKeyArn}</kmsKeyArn>
+                    <layers>{$ctx:layers}</layers>
+                    <memorySize>{$ctx:memorySize}</memorySize>
+                    <publish>{$ctx:publish}</publish>
+                    <role>{$ctx:role}</role>
+                    <runtime>{$ctx:runtime}</runtime>
+                    <tags>{$ctx:tags}</tags>
+                    <timeout>{$ctx:timeout}</timeout>
+                    <mode>{$ctx:mode}</mode>
+                    <securityGroupIds>{$ctx:securityGroupIds}</securityGroupIds>
+                    <subnetIds>{$ctx:subnetIds}</subnetIds>
+                </amazonlambda.createFunction>
+                <respond/>
+            </inSequence>
+            <outSequence/>
+            <faultSequence/>
+        </resource>
+    </api> 
+    ```
+        
+2. Right-click on the Composite Application Project and click on **Export Project Artifacts and Run**. Select **Run on Micro Integrator**.
+3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+{! /references/connectors/exporting-artifacts.md !}
```

Comment:
Please check format. This doesn't render.

Reply:
Done.